app/views/user_mailer/email_notify.html.erb

```diff
@@ -0,0 +1,5 @@
+<p><%= @user.name %></p>
```

Comment:
Rather than hardcoding your emails, you could instead use the Template Engine provided by SendGrid and pass a template ID on your API calls. 

To use templates, you will need the Templates APP on and to pass the [X-SMTPAPI header for the template ID](https://sendgrid.com/docs/API_Reference/SMTP_API/apps.html#templates)